tests/tracer/test_ext.py

```diff
@@ -23,5 +26,118 @@ def _updateenv(monkeypatch, env):
 
 @pytest.mark.parametrize("name,environment,tags", _ci_fixtures())
 def test_ci_providers(monkeypatch, name, environment, tags):
+    """Make sure all provided environment variables from each CI provider are tagged correctly."""
     _updateenv(monkeypatch, environment)
-    assert tags == ci.tags(environment), "wrong tags in {0} for {1}".format(name, environment)
+    extracted_tags = ci.tags(environment)
+    for key, value in tags.items():
+        assert extracted_tags[key] == value, "wrong tags in {0} for {1}".format(name, environment)
+
+
+def test_git_extract_user_info_author():
+    """Make sure that git commit author name, email, and date are extracted and tagged correctly."""
+    expected_author = ("John Doe", "john@doe.com", "2021-02-19T08:24:53Z")
+    expected_call_format = ["git", "show", "-s", "--format=%an,%ae,%ad", "--date=format:%Y-%m-%dT%H:%M:%S%z"]
+    mock_author_output = b"John Doe,john@doe.com,2021-02-19T08:24:53Z"
```

Comment:
what happens if one of these values is not available?

e.g. can we get `b",john@doe.com,<date>"` ?

Reply:
I've done some reading up on git documentation, but it looks like we should always get the three values from `git show -s ...`. From https://git-scm.com/docs/git-show, `git show ` defaults to showing the HEAD commit and should always have the three values available.